README.md

```diff
@@ -0,0 +1,80 @@
+# OpenTelemetry Instrumentation: Spring and SpringBoot
+
+This package streamlines the manual instrumentation process in OpenTelemetry for Spring and SpringBoot. It will enable you to add traces to requests, and database calls with minimal changes to application code. This package will not fully automate your OpenTelemetry instrumentation, instead, it will provide you with better tools to instrument your own code.
+This contribution to OpenTelemetry will follow in the footsteps of the existing Spring integration in [OpenCensus](https://github.com/census-instrumentation/opencensus-java/tree/master/contrib/spring/src/main/java/io/opencensus/contrib/spring).
+
+This starter guide contains three tutorials. The first tutorial will walk you through span creation and propagation using the JavaSDK and the HTTP client RestTemplate. 
```

Comment:
Do you need a new section for general configuration / dependencies of Spring and OT even for those users who do not use your added support? 
i.e., for people who want to use manual API. .

Reply:
I included a section for general configurations in the full read me. I kept this one to only include my annotations and my feature proposal.